MyToolsYourToolsClient/src/app/admin-panel/notifications/notifications.component.ts

```diff
@@ -77,10 +77,22 @@ export class NotificationsComponent implements OnInit {
       message: this.inputText[requestId],
       ratedUserId: currentRequest.targetUserId,
       ratingUserId: currentRequest.ownerId
-    }
-    this.opinionService.addOpinion(tmpOpinion).subscribe();
-    this.opinions = this.opinions.filter(n => n.id !== requestId);
-    this.notificationService.deleteNotification(requestId).subscribe();
+
+  } 
+  
+
+    this.opinionService.addOpinion(tmpOpinion).subscribe(
+      result => {
+        this.alertService.success('Opinia została przesłana');
+        this.opinions = this.opinions.filter(n => n.id !== requestId);
+        this.notificationService.deleteNotification(requestId).subscribe();
```

Comment:
tutaj jeszcze przydałaby się uaktualnić punkty na navbarze czyli this.userService.announceUserUpdate(true). Bo po dodaniu opinii zmieniamy liczbę punktów.